claragenomics/dl4atac/train/metrics.py

```diff
@@ -86,11 +87,14 @@ def __init__(self):
     def __call__(self, x, y):
         x = Metric.convert_to_tensor(x)
         y = Metric.convert_to_tensor(y)
-        x_norm = x - torch.mean(x)
-        y_norm = y - torch.mean(y)
-
-        self.val = torch.sum(x_norm * y_norm) / (torch.sqrt(torch.sum(x_norm**2))
-                                             * torch.sqrt(torch.sum(y_norm**2)))
+        xm = x.mean()
+        ym = y.mean()
+        x = x - xm
+        y = y - ym
+        #x.add_(-1*xm)
```

Comment:
let's remove the commented lines of code. feel free to resolve this once the change is made